lib/lucidboard/twiddler.ex

```diff
@@ -130,7 +130,17 @@ defmodule Lucidboard.Twiddler do
         )
       end
 
-    Repo.paginate(query, page: page_index)
+    filter_query =
+      if boards_filter == "created" do
+        from([base, user: u, role: r] in query,
+          where:
+            r.user_id == ^user_id
+        )
+      else
+        query
+      end
+
+    Repo.paginate(filter_query, page: page_index)
```

Comment:
wow, multilne comments.. 
i couldn't figure out how to work this in the query above, maybe there is a cleaner way?